stactools_planet/stactools/planet/planet_item.py

```diff
@@ -1,17 +1,29 @@
+import os
 from copy import deepcopy
 import json
 import logging
 
 import fsspec
 import pystac
+from pystac.link import Link
 from pystac.utils import (str_to_datetime, make_absolute_href)
+from pystac.extensions.eo import Band
 from shapely.geometry import shape
 
 from stactools.planet import PLANET_PROVIDER
 from stactools.planet.constants import PLANET_EXTENSION_PREFIX
 
+from osgeo import gdal
```

Comment:
I'd lightly suggest using `rasterio` instead, and that would keep the `planet` module in line with the other `stactools` modules: https://github.com/stac-utils/stactools/search?q=rasterio

Reply:
I will take care of that and add the changes to the PR